samples/matter/lock/sample.yaml

```diff
@@ -40,3 +40,10 @@ tests:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+  sample.matter.lock.thread_wifi_switched_ext_patch:
```

Comment:
@doublemis1 what do you think about that regarding our tests? From my perspective, I'm ok with both options.

Reply:
Does it means that the flash procedure or OTA procedure will be different?

Reply:
The flash procedure will be the same, the DK gets up 2 seconds longer, and the OTA and BLE DFU procedures may take more time because we need to update an additional image. Moreover, we cannot perform firmware updates from the previous solution to the new one because the new partition breaks compatibility. So, there is my consideration of how we want to test it.

Reply:
there should be not problem with that, however updating the wifi patch by BLE SMP DFU will be not covered since we need to send third image